client/me/developer/style.scss

```
@@ -0,0 +1,34 @@
+@import "@wordpress/base-styles/breakpoints.scss";
+
+.developer__is_dev_account-fieldset-is-loading, {
+	.components-form-toggle {
+		> * {
```

```suggestion
		.components-form-toggle__thumb {
```

It's not a good practice to use `*` selector, otherwise it will be harder to do maintenance and some nodes can get such styles unintentionally. It's better to be more specific - using classes and specify which exactly nodes should be visible/hidden.

---

I simply took what we had in `profile/style.scss` and didn't put much thought into improving the css :p Thanks for the suggestion!